patches/openwrt/0018-imagebuilder-hacky-way-to-use-ib-for-gluon.patch

```diff
@@ -0,0 +1,46 @@
+From: lemoer <git@irrelefant.net>
+Date: Wed, 6 Jan 2021 05:41:56 +0100
+Subject: imagebuilder: hacky way to use ib for gluon
+
+- Include scripts/ into the imagebuilder.
+- Install openwrt/staging_dir/hostpkg into the imagebuilder.
+- Install openwrt/staging_dir/hostpkg/lib/lua into the imagebuilder.
+
+diff --git a/target/imagebuilder/Makefile b/target/imagebuilder/Makefile
+index 05c7cf3318deb18af1851524c533b1c47af52e1e..e478b5f266cf47a638c82798e8ee596d1820d2ee 100644
+--- a/target/imagebuilder/Makefile
++++ b/target/imagebuilder/Makefile
+@@ -35,6 +35,11 @@ $(BIN_DIR)/$(IB_NAME).tar.xz: clean
+ 		$(TMP_DIR)/.packageinfo \
+ 		$(PKG_BUILD_DIR)/
+ 
++	mkdir -p $(PKG_BUILD_DIR)/gluon
++	$(CP) -L \
++		$(TOPDIR)/../scripts \
++		$(PKG_BUILD_DIR)/gluon/
```

Comment:
I suppose, this will become unnecessary, since only `check_site.lua` was used and now (on the master branch) it is built into `gluon-core`. But this should be cross checked.

Reply:
Done.